schedule/yast/transactional_server/create_hdd_transactional_server.yaml

```diff
@@ -29,33 +29,18 @@ conditional_schedule:
         BACKEND:
             svirt:
             - shutdown/svirt_upload_assets
-    license_reg_scc_addon:
-        DISTRI:
-            sle:
-            - installation/accept_license
-            - installation/scc_registration
-            - installation/addon_products_sle
-    online_repos_inst_mode_logpackes:
-        DISTRI:
-            opensuse:
-            - installation/online_repos
-            - installation/installation_mode
-            - installation/logpackages
-    user_settings_root:
-        DISTRI:
-            sle:
-            - installation/user_settings_root
 schedule:
     - installation/bootloader_start
     - installation/welcome
-    - "{{license_reg_scc_addon}}"
-    - "{{online_repos_inst_mode_logpackes}}"
+    - installation/accept_license
```

Comment:
What is the motivation of this change?
I also would ask to put it in a separate commit.

Reply:
Explained in person ;)